### ISSUE TITLE
Is it possible to fine-tune groundingdino without a validation set?

### ISSUE THREAD

**Issue by TianrunUOM** [IMPACT-L/GroundingDINO-US-SAM#2]
I look forward to your reply—many thanks in advance.
